consensus/src/simplex/actors/batcher/actor.rs

```diff
@@ -130,7 +130,7 @@ impl<
                         self.reporter
                             .report(Activity::Notarize(notarize.clone()))
                             .await;
-                        self.notarizes[index as usize] = Some(notarize.clone());
+                        self.notarizes.push(notarize.clone());
```

Comment:
I kinda regret calling this `push` because the item is added to the signer's index 🤔

Reply:
Yeah I think you did that for compatibility reasons but I agree it's a bit strange now in hindsight

Reply:
Yeah, that would be my recommendation ("key" is inferred from message basically)